### PR TITLE
Split minikube install task into tabs (per-OS)

### PR DESCRIPTION
**Pull request by sftim**
Relevant to #12852 

I saw some issues when I tried to use an unordered list inside the tabs, so I worked around that using Unicode bullets.

Even if this isn't perfect, I think it's better than the current page and so is worth merging.

### CODE REVIEW

File: content/en/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This page shows you how to install [Minikube](/docs/tutorials/hello-minikube), a
 
 {{% capture prerequisites %}}
 
-VT-x or AMD-v virtualization must be enabled in your computer's BIOS.  To check this on Linux run the following and verify the output is non-empty:
+VT-x or AMD-v virtualization must be enabled in your computer's firmware settings.  To check this on Linux, run the following command and verify the output is non-empty:
 ```shell
 egrep --color 'vmx|svm' /proc/cpuinfo
```
Comment (kbhawkey) [inline content/en/docs/tasks/tools/install-minikube.md]: @sftim , in general, introducing the tabs per OS nicely cleans up the page. I also found that the formatting for the content in tabs does some unexpected rendering.
nit: This prerequisite section applies to all, but yet the user is instructed to verify only on Linux.

Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: I'll move the section about verifying virtualization to the Linux section, and I'll see if I give a similar advice for Windows.

I (or anyone else who's keen) might tackle the cleanup section in a separate PR.

Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: Updated the verification steps. @kbhawkey how does this look to you?

File: content/en/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -106,6 +115,13 @@ sudo mv minikube /usr/local/bin
 
 {{% /tab %}}
 {{% tab name="Windows" %}}
+
+You'll need to use a computer with hardware virtualization enabled. You can
+use `systeminfo` to [check](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/reference/hyper-v-requirements#verify-hardware-compatibility) whether your PC has this support.
+Although `systeminfo` mentions about Hyper-V, the status of hardware support
+for virtualization is also applicable to VirtualBox and other hypervisor
+applications.
+
```
Comment (kbhawkey) [inline content/en/docs/tasks/tools/install-minikube.md]: Hi, @sftim. Just read through again. Looks good 😄. Do you think the virtualization paragraphs should be below the `kubectl` instructions, and with the hypervisor links?
Would be great to get a `minikube` owner to review.

Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: >  Would be great to get a minikube owner to review.
I agree! I've moved the wording around, so I'm reasonably confident that the changed documentation remains correct, but in truth I've never installed Minikube myself.

Comment (kbhawkey) [reply content/en/docs/tasks/tools/install-minikube.md]: OK. I used this content as a refresher for Windows, but ended up consulting:
https://github.com/kubernetes/minikube#installation. Could try and ping the minikube channel on Slack?

File: content/en/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -9,87 +9,137 @@ card:
...
+• [HyperKit](https://github.com/moby/hyperkit)
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+• [VMware Fusion](https://www.vmware.com/products/fusion)
+
 
 {{< note >}}
-This document shows you how to install Minikube on Linux using a static binary. For alternative Linux installation methods, see [Other Ways to Install](https://github.com/kubernetes/minikube#other-ways-to-install) in the official Minikube GitHub repository.
+If you have a different hypervisor installed, you _may_ need to disable or remove it first.
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: Oops! Good spot.

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: I guess we can assume people are using modern Mac computers, and won't have (eg) an old version of Parallels installed.

File: content/en/docs/tasks/tools/install-minikube.md [hunk 4]
```diff
@@ -9,87 +9,137 @@ card:
...
-curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 \
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
   && chmod +x minikube
 ```
 
 Here's an easy way to add the Minikube executable to your path:
 
 ```shell
-sudo mv minikube /usr/local/bin
+sudo cp minikube /usr/local/bin && rm minikube
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: Makes sense. I'd prefer to leave that change for another PR.

**Comment by kbhawkey** [comment]
@sftim , with the latest changes (tabs/OS), the top TOC seems confusing.
Suggested change:
- `Before you begin` prereq section could include: Install kubectl and Install a Hypervisor in a tab set/OS
- `Install minikube` A second tab set could contain the instructions for the minikube install/OS
- `Troubleshooting the minikube installation`
- `What's next`

**Comment by bradtopol** [comment]
/assign @steveperry-53 
@steveperry-53 can you review this one and see if you like the new proposed structure?

**Comment by sftim** [comment]
Doubt I'll have time to work on this before 2019-05-09T2100 (UTC)

**Comment by sftim** [comment]
(Rebased)

File: content/en/docs/tasks/tools/install-minikube.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tasks/tools/install-minikube.md]: Done, anyway

**Comment by sftim** [comment]
(Rebased)

**Pull request merged**